CONTRIBUTING.md

```diff
@@ -312,6 +312,53 @@ load the data. Right now we don't provide any instructions inside here yet, but
 [here's how the official ReasonML playground did
 it](https://github.com/reasonml/reasonml.github.io/blob/source/website/setupSomeArtifacts.js#L65).
 
+### Upgrading the Reason version
```

Comment:
@ryyppy  Thanks for this work documenting the process! One question / suggestion :)

I started reading this section as part of the larger "Contributing to the BS Playground Bundle" above, but got confused when 2 versions of refmt are discussed. My main doubt was "why does the BS playground need two versions of refmt? Wouldn't refmt_api be enough?".

Then I understood (maybe wrongly?) that `refmt_api` is what is used in the playground, and `refmt_main3` is what BS uses when it runs natively, so it's not used in the playground (that was the confusing part).

If that assumption is right, maybe it'd be worth making the whole "Upgrading the Reason version" section an h2 (##) instead of h3, so that it "escapes" the "Contributing to the BS Playground Bundle" above and stands on its own, as an explanation on how these 2 packed files are generated, and how to keep Reason versions in sync with BS.

Maybe also rename the section to "Upgrading the Reason version for both native- and JavaScript-compiled versions of BS".

I hope that makes sense.